pai-management/src/gpu-exporter/dockerfile

```diff
@@ -29,17 +29,20 @@ ENV PATH=$PATH:$NV_DRIVER/bin
 WORKDIR /root/
 
 RUN apt-get update && \
-    apt-get -y install wget build-essential python python-pip git pciutils
+    apt-get -y install wget build-essential python python-pip git pciutils iftop lsof
 
 COPY copied_file/exporter/* /usr/local/
 
 RUN wget https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-ce.tgz
 RUN cp docker-17.06.2-ce.tgz /usr/local
 RUN tar xzvf /usr/local/docker-17.06.2-ce.tgz -C /usr/local/
 RUN cp -r /usr/local/docker/* /usr/bin/
-
+RUN git clone https://github.com/yadutaf/infilter
+RUN cd infilter && make && cd ..
+RUN cp infilter/infilter /usr/bin
 #
 # start
-#
-
-CMD python /usr/local/job_exporter.py /datastorage/prometheus 30
+#  
+# The start cmd will be called at K8s config yaml file.
+# local container test cmd example: 
+# CMD python /usr/local/job_exporter.py /datastorage/prometheus 30
```

Comment:
As you've changed to start it with k8s yaml, this line seems duplicate

Reply:
for remind developer local test docker container cmd. If test doesn't want to submit to k8s, then it could run this container at local host.